pkg/kube/configmap/configmap_builder.go

```diff
@@ -42,6 +42,13 @@ func (b *builder) SetLabels(labels map[string]string) *builder {
 	return b
 }
 
+func (b *builder) SetData(data map[string]string) *builder {
```

Comment:
optional: small inconsistency in naming SetField / SetData. What do you think about one of the pairs SetField / SetFields or SetDataField / SetData?

Reply:
good point, have renamed SetFiled to SetDataField